addons/mail/static/src/models/dialog/dialog.js

```diff
@@ -19,15 +19,26 @@ function factory(dependencies) {
             if (this.followerSubtypeList) {
                 return replace(this.followerSubtypeList);
             }
+        },
+        _computeComponentName() {
+            if (this.attachmentViewer) {
+                return 'AttachmentViewer';
+            }
+            if (this.followerSubtypeList) {
+                return 'FollowerSubtypeList';
+            }
+            return '';
```

Comment:
This should be clear instead of empty string, especially important if the field is made required.